base/index.mustache

```diff
@@ -10,23 +10,23 @@ app.use(bodyParser.json());
 app.use(cors());
 
 fs.readdirSync('./middleware').forEach(name => {
-  app.use(require('./middleware/' + name));
+  app.use(require(`./middleware/${name}`));
 });
 
 fs.readdirSync('./routes').forEach(name => {
-  const route = require('./routes/'+name);
+  const route = require(`./routes/${name}`);
   app.use(route.http);
   route.ws(io);
 });
 
-app.use(function (err, req, res, next) {
+app.use((err, req, res) => {
```

Comment:
I'm going to remove the rule and reset some of the changes I made to accomodate it.